### PR TITLE
Added Javascript Menu configuration in Portfolio Project

### PR DESCRIPTION
**Pull request by ohyajk**
I added Navbar Menu javascript configuration with my coding partner.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,20 +9,22 @@
...
-                <img class="menu" src="./assets/ic_menu.svg" alt="hamburger menu">
-                <ul>
+                <img class="menu" id="hamburger" src="./assets/ic_menu.svg" alt="hamburger menu">
+                <ul id="ul">
                     <li><a class="underline" href="#Home">Home</a></li>
                     <li><a href="#project">Portfolio</a></li>
                     <li><a href="#about">About</a></li>
                     <li><a href="#contact">Contact</a></li>
                     <div class="mob-abs-2"></div>
+                    <img id="cross" class="menu-cross" src="./assets/cross.svg" alt="cross">
```
Comment (youmari) [inline index.html]: - [x] Your menu is great but it's just a tiny change you should position this box below as in the Figma design in order to match the Figma design 
- box 
Screenshot 2022-11-14 at 15 11 34

- Yours 
Screenshot 2022-11-14 at 15 11 40
- Figma 
Screenshot 2022-11-14 at 15 11 50

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @ohyajk , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -9,20 +9,23 @@
...
                     <li><a class="underline" href="#Home">Home</a></li>
                     <li><a href="#project">Portfolio</a></li>
                     <li><a href="#about">About</a></li>
                     <li><a href="#contact">Contact</a></li>
+                    <div class="mob-abs-dot"></div>
                     <div class="mob-abs-2"></div>
+                    <img id="cross" class="menu-cross" src="./assets/cross.svg" alt="cross">
                 </ul>
             </nav>
         </header>
```
Comment (HAMMAS-SALEEM) [inline index.html]: Capture

- [ ] When i click on any of the nav link in the desktop version then they work as expected but after clicking these link get vanish. Kindly fix this issue 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @ohyajk :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github Flow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ohyajk) [reply index.html]: You are not reviewing the mobile menu. Please keep your focus on a given task. You are talking about optional features and wasting my review.

additional

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @ohyajk , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**